js/main.js

```diff
@@ -402,3 +403,15 @@ function isDone(directions) {
   // If there are no departures and they all have isDone == true, we're done
   return true;
 }
+
+function checkOutdatedSource(){
+  $.ajax({
+    url: 'https://api.github.com/repos/umts/BusInfoBoard',
```

Comment:
So, its worth mentioning that its *possible* that people are deploying this software using some other method than GH pages, or even deploying forks of the repo, etc.  The README says, "Alternatively, you can host a copy of the code on any web server capable of serving static files, or even keep a copy of the code as a local file on your kiosk device."